spec/pushmi_pullyu/logging_spec.rb

```diff
@@ -28,8 +28,8 @@ class LoggerTest
     expect(PushmiPullyu::Logging.logger).to have_received(:debug).with(an_instance_of(String)).once
   end
 
-  describe '#reopen' do
-    let!(:tmp_dir) { 'tmp/test_dir' }
+  describe '.reopen' do
```

Comment:
`#` is for instance level methods. Most of these are class/module level methods so should be using `.`

http://www.betterspecs.org/#describe